src/MonologCreator/Processor/ExtraFieldProcessor.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace MonologCreator\Processor;
```

Comment:
There should be no empty newline between the opening php tag and the first code line.